### ISSUE TITLE
Hello!

### ISSUE THREAD

**Issue by mattmassicotte** [codeface-io/LSPService#5]
I know that we tried, and totally failed, to collaborate. But there have been developments and I just wanted to check in.

(and sorry for doing it this way, is there a better way for me to get in touch?)

**Comment by codeface-io** [comment]
@mattmassicotte Hey Matt 😃

No worries at all! If a potential collaboration was short of anything, it was my consistent effort.

Open-source was not a top priority for me in recent months, the main reason being a new full-time job. Even though merging our LSP code would be a bit of a tall order right now, I'd still appreciate staying in touch and exchanging knowledge. Feel free to say hello via sebastian@codeface.io. We could also set up a chat, that would be fun.

Thanks for letting me know of that new trace back on August 23rd. Somehow I managed not to react, my bad 🙈 I read that there's one place in macOS where a sandboxed app is allowed to run scripts. Without experimenting or totally thinking this through, my hunch was that this cannot help to simplify the user experience or development of Codeface, but I could be wrong.

I tried hard, for months, to make the LSPService unnecessary. I also approached Marcin, but he didn't react on any channel. So my current plan is to at least make the LSPService simpler and more general, which would allow setting up the servers via a GUI from within Codeface ...

Cheers,
Seb
